### PR TITLE
Translate 819 822 runtime upgrade

### PR DESCRIPTION
**Pull request by wuzhong-papermoon**
Original PR：
https://github.com/moonbeam-foundation/moonbeam-docs/pull/822
https://github.com/moonbeam-foundation/moonbeam-docs/pull/819

### CODE REVIEW

File: builders/build/runtime-upgrades.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+## 序言 {: #introduction }
+
+Moonbeam的runtime升级允许网络在不需要硬分叉的情况下维护/改变链逻辑。 这些runtime升级可以引入新功能，提高性能，修复错误并响应不断变化的要求。
+
+本页面为每个基于Moonbeam的网络按区块提供历史runtime升级记录。
+
+## Runtime升级 {: #runtime-upgrades-by-block }
+
+下表列出了不同网络上的每个runtime升级，以及它们发生时的区块号。 Runtime升级首先在Moonbase Alpha上发生，然后分别发布到Moonriver和Moonbeam上。您可以在[GitHub Moonbeam发布页面](https://github.com/moonbeam-foundation/moonbeam/releases){target=_blank}获取每个Runtime的发布公告。
```
Comment (eshaben) [inline builders/build/runtime-upgrades.md:16]:
```suggestion
下表列出了不同网络上的每个runtime升级，以及它们发生时的区块号。 Runtime升级首先在Moonbase Alpha上发生，然后分别发布到Moonriver和Moonbeam上。您可以在[GitHub Moonbeam发布页面](https://github.com/moonbeam-foundation/moonbeam/releases){target=\_blank}获取每个Runtime的发布公告。
```

File: builders/build/runtime-upgrades.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+|  2403   | [4163078](https://moonbeam.subscan.io/block/4163078){target=_blank} | [4770488](https://moonriver.subscan.io/block/4770488){target=_blank} | [4804425](https://moonbase.subscan.io/block/4804425){target=_blank} |
+|  2500   |                                  -                                  | [5175574](https://moonriver.subscan.io/block/5175574){target=_blank} | [5053547](https://moonbase.subscan.io/block/5053547){target=_blank} |
+|  2501   | [4543267](https://moonbeam.subscan.io/block/4543267){target=_blank} | [5211264](https://moonriver.subscan.io/block/5211264){target=_blank} | [5194594](https://moonbase.subscan.io/block/5194594){target=_blank} |
+|  2601   |                                  -                                  |                                  -                                   | [5474345](https://moonbase.subscan.io/block/5474345){target=_blank} |
+|  2602   | [4977160](https://moonbeam.subscan.io/block/4977160){target=_blank} | [5638536](https://moonriver.subscan.io/block/5638536){target=_blank} | [5576588](https://moonbase.subscan.io/block/5576588){target=_blank} |
```
Comment (eshaben) [inline builders/build/runtime-upgrades.md:76]:
```suggestion
|  2602   | [4977160](https://moonbeam.subscan.io/block/4977160){target=\_blank} | [5638536](https://moonriver.subscan.io/block/5638536){target=\_blank} | [5576588](https://moonbase.subscan.io/block/5576588){target=\_blank} |
|  2700   | [5504531](https://moonbeam.subscan.io/block/5504531){target=\_blank} | [6041969](https://moonriver.subscan.io/block/6041969){target=\_blank} | [5860584](https://moonbase.subscan.io/block/5860584){target=\_blank} |
|  2801   |                                  -                                   |                                   -                                   | [6209638](https://moonbase.subscan.io/block/6209638){target=\_blank} |

```

add the missing runtimes

**Review by eshaben — CHANGES_REQUESTED** [review]
Sweeeeet! Can you please make sure to switch all of the `target=_blank`s in the table to `target=\_blank`s ?

File: builders/build/runtime-upgrades.md (hunk 2 above)
Comment (eshaben) [inline builders/build/runtime-upgrades.md:76]:
```suggestion
| [2601](https://forum.moonbeam.network/t/runtime-rt2600-schedule/1372/5){target=\_blank}  |                                  -                                   |                                   -                                   | [5474345](https://moonbase.subscan.io/block/5474345){target=\_blank} |
| [2602](https://forum.moonbeam.network/t/runtime-rt2600-schedule/1372/13){target=\_blank} | [4977160](https://moonbeam.subscan.io/block/4977160){target=\_blank} | [5638536](https://moonriver.subscan.io/block/5638536){target=\_blank} | [5576588](https://moonbase.subscan.io/block/5576588){target=\_blank} |
| [2700](https://forum.moonbeam.network/t/runtime-rt2700-schedule/1441/3){target=\_blank}  | [5504531](https://moonbeam.subscan.io/block/5504531){target=\_blank} | [6041969](https://moonriver.subscan.io/block/6041969){target=\_blank} | [5860584](https://moonbase.subscan.io/block/5860584){target=\_blank} |
| [2801](https://forum.moonbeam.network/t/runtime-rt2801-schedule/1616/4){target=\_blank}  |                                  -                                   |                                   -                                   | [6209638](https://moonbase.subscan.io/block/6209638){target=\_blank} |
```

Comment (wuzhong-papermoon) [reply builders/build/runtime-upgrades.md]: There are quite a few so I made the changes in codespace.

Comment (eshaben) [reply builders/build/runtime-upgrades.md]: Ok, thanks. Can you please add the links for 2601, 2602, 2700, and 2801 to the forum, as shown in my suggestion above?

**Pull request closed without merging**